src/navigation/sectionalTabs/sectionalTabs.jsx

```diff
@@ -60,11 +64,12 @@ const propTypes = {
 };
 
 const defaultProps = {
-    items: [],
-    selectedTabKey: undefined,
-    resizeThrottle: 100,
     beforeChange: undefined,
+    classes: undefined,
```

Comment:
Please use `null` instead of `undefined` for `defaultProps`.

Reply:
done